code/config/config_nlweb.yaml

```diff
@@ -31,6 +31,12 @@ decontextualize_enabled: true
 # When set to false, the system will not check if required information is present before processing queries
 required_info_enabled: true
 
+# Headers for HTTP requests
+headers:
+  # User-Agent header
+  user_agent: "NLWeb/1.0"
+  data_retention: "Data will be retained for 30 minutes"
+
```

Comment:
What's the source for this claim?  Can we make this promise not knowing what model the end user is using?